client/components/data/query-post-likes/index.jsx

```diff
@@ -1,67 +1,49 @@
 import PropTypes from 'prop-types';
-import { Component } from 'react';
-import { connect } from 'react-redux';
-import { Interval } from 'calypso/lib/interval';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useInterval } from 'calypso/lib/interval';
 import { requestPostLikes } from 'calypso/state/posts/likes/actions';
 import { getPostLikeLastUpdated } from 'calypso/state/posts/selectors/get-post-like-last-updated';
 import { getPostLikes } from 'calypso/state/posts/selectors/get-post-likes';
 
-class QueryPostLikes extends Component {
-	static propTypes = {
-		siteId: PropTypes.number.isRequired,
-		postId: PropTypes.number.isRequired,
-		needsLikers: PropTypes.bool,
-		hasPostLikes: PropTypes.bool,
-		maxAgeSeconds: PropTypes.number, // max age of likes data in milliseconds
-		lastUpdated: PropTypes.number, // timestamp of when the like data was last updated
-		requestPostLikes: PropTypes.func.isRequired,
-	};
+const MAX_AGE_MS = 120 * 1000;
```

Comment:
this one used to be a prop but it's not set by any of the usages so I'm removing it from the public API